lang/luajit/Makefile

```diff
@@ -0,0 +1,80 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luajit
+PKG_VERSION:=2.1.0-beta2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=git://github.com/LuaJIT/LuaJIT.git
+PKG_SOURCE_VERSION:=a7bec69a7593b2887a6d1d8dd13272aa005c36fa
+PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
Then `PKG_VERSION` had better be set to a value to reflect this situation, e.g. `devel-2016-06-xx-a7bec...`.  Another option would be to backport the fix into a release/beta version.